src/main/java/uk/gov/hmcts/reform/hmi/models/ApiResponse.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.hmi.models;
+
+import lombok.Data;
+
+@Data
+public class ApiResponse {
+
+    private Integer statusCode;
+
+    private String message;
+
+    public ApiResponse(Integer statusCode, String message) {
+        this.statusCode = statusCode;
+        this.message = message;
+    }
```

Comment:
We can add the @AllArgsConstructor and remove lines 12 - 15 :)

Reply:
fixed. thanks.